### ISSUE TITLE
Make 1click install options for Digital Ocean, Azure, and AWS

### ISSUE THREAD

**Issue by jackzampolin** [blockstack/blockstack-core#534]
- [ ] AWS 
  * Blockers: Need access to Amazon Marketplace for my login
- [ ] DO
  * Blockers: Need access to DO account
- [x] Azure
  * Blockers: Need access to Azure Marketplace listing

**Comment by jackzampolin** [comment]
@muneeb-ali ☝️

**Comment by jackzampolin** [comment]
DO has no public process for contributing 1 click applications. I've got a support ticket into them.

**Comment by jackzampolin** [comment]
Running into a CORS issue with https://github.com/OB1Company/deploy. Got on the phone with the Dev from OB and he said hes had a ticket open with DO for a couple of weeks and no response. I've also tried a couple of the forks with no luck. Pitched in on an open ticket with OB to DO.

**Comment by jackzampolin** [comment]
Currently going back and forth with Azure support on the right way to upload image to Azure. Still don't have access to our AWS seller portal.

**Comment by jackzampolin** [comment]
Done with the Azure listing. Still blocked on AWS and DO.

**Comment by jackzampolin** [comment]
While we do need some better deployment documentation for Blockstack-Core, I'm going to close this as too specific and out dated and open another issue to track this work when we pick it back up.
